functests/1_performance/performance.go

```diff
@@ -192,13 +194,55 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 		})
 
 		It("[test_id:35363][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running on the host", func() {
-			Skip("until bugs https://bugzilla.redhat.com/show_bug.cgi?id=1912118 and https://bugzilla.redhat.com/show_bug.cgi?id=1903302 fixed")
+			By("Checking minimal required cluster version")
+			clusterVersion := ""
+			cv := &v1.ClusterVersion{}
+			key := types.NamespacedName{
+				Name: "version",
+			}
+			err := testclient.Client.Get(context.TODO(), key, cv)
+			Expect(err).ToNot(HaveOccurred(), "cannot find the Cluster version object "+key.String())
+			clusterVersion = cv.Status.Desired.Version
+			splitClusterVersion := strings.Split(clusterVersion, ".")
+			Expect(len(splitClusterVersion) > 2).Should(BeTrue(), "Cluster version in X.Y.Z format cannot be found")
+			zStream, err := strconv.Atoi(splitClusterVersion[2])
```

Comment:
I think CI release version can be different from the expected on because it built from master, so to debug e2e, you can print the version here.

Reply:
ack. notice that this is 4.7